sw360-single/sw360-install.sh

```diff
@@ -177,27 +175,25 @@ if $doClean; then
     mvn clean
 fi
 
-if $doLibs; then
-    mvn install $mavenParameters
-fi
-
-if $doBackend; then
-    echo "Start of backend deployment of sw360"
-    cd $wd/backend
-    mvn tomcat7:redeploy-only $mavenParameters
-fi
-
-if $doFrontend; then
-    echo "Start of frontend deployment of sw360"
-    cd $wd/frontend
-    mvn install -P deploy $mavenParameters
-elif $doPortlets; then
-    echo "Start of frontend deployment of sw360"
-    cd $wd/frontend/sw360-portlet
+if $doAll; then
     mvn install -P deploy $mavenParameters
+else
```

Comment:
I though, one could choose to do the libs and the frontend but no backend.

I thing nobody will miss this feature.